backends/kv/writer.go

```diff
@@ -89,6 +89,14 @@ func (kv *Backend) Write(request *frames.WriteRequest) (frames.FrameAppender, er
 		case frames.ErrorIfTableExists:
 			return nil, fmt.Errorf("table '%v' already exists; either use a differnet save mode or save to a different table", tablePath)
 		}
+	} else {
+		exists, err := checkPathExists(tablePath, container)
+		if err != nil {
+			return nil, err
+		}
+		if exists {
+			return nil, fmt.Errorf("folder '%v' already exists; you can't write to an exisitng folder without a schema file", tablePath)
```

Comment:
exisitng -> existing

Reply:
To avoid confusion:
without a schema file -> unless it already contains a schema file